pkg/plugins/diff/diff.proto

```diff
@@ -0,0 +1,39 @@
+/*
+Run from lakeFS root:
+    protoc --proto_path=pkg/plugins/diff --go_out=pkg/plugins/diff --go_opt=paths=source_relative \
+        --go-grpc_out=pkg/plugins/diff --go-grpc_opt=paths=source_relative diff.proto
+ */
+syntax = "proto3";
+package diff;
+option go_package = "github.com/treeverse/lakefs/plugins/diff";
+
+message DiffRequest {
+  TablePaths paths = 1;
+  GatewayConfig gateway_config = 2;
+}
+
+message TablePaths {
```

Comment:
push the definition up - before we use it.

Reply:
done